### PR TITLE
det meste af Item package implementeret

### PR DESCRIPTION
**Pull request by AndreasBoegh**
under powerUpItem mangler update og startBuff

### CODE REVIEW

File: src/hospitalgame/item/Inventory.java [hunk 1]
```diff
@@ -31,54 +33,75 @@ public Inventory(int maxWeight) {
 
     /**
      * adds an item to the inventory
+     *
      * @param item is the item object to be added to the inventory.
      * @return returns true if the item has been added
      */
     public boolean addItem(Item item) {
```
Comment (frederikrosenberg) [inline src/hospitalgame/item/Inventory.java:38]: Is not checking if there is space for item in the inventory

**Review by frederikrosenberg — CHANGES_REQUESTED** [review]
See addItem in inventory

File: src/hospitalgame/item/Inventory.java [hunk 2]
```diff
@@ -31,54 +33,78 @@ public Inventory(int maxWeight) {
...
-        throw new NotImplementedException();
+        if (item.getWeight() + getTotalWeight() <= maxWeight) {
+            return inventory.add(item);
+        } else { return false; }
```
Comment (frederikrosenberg) [inline src/hospitalgame/item/Inventory.java]: Please do a format (Source -> format) i NetBeans

**Review by frederikrosenberg — CHANGES_REQUESTED** [review]

**Review by frederikrosenberg — APPROVED** [review]

File: src/hospitalgame/item/Inventory.java [hunk 3]
```diff
@@ -31,54 +31,80 @@ public Inventory(int maxWeight) {
...
-        throw new NotImplementedException();
+        return inventory.remove(item);
     }
 
     /**
      * prints items from the inventory to console
      */
     public void showItems() {
-
+        for (int i = 0; i <= inventory.size(); i++) {
```
Comment (larsbjn) [inline src/hospitalgame/item/Inventory.java]: Maybe a for each loop?

File: src/hospitalgame/item/Inventory.java [hunk 4]
```diff
@@ -31,54 +31,80 @@ public Inventory(int maxWeight) {
...
     /**
      * returns the totalWeight of the items currently in the inventory
-     * @return int
+     *
+     * @return the total weight in the inventory
      */
     public int getTotalWeight() {
-        throw new NotImplementedException();
+        int totalWeight = 0;
+        for (int i = 0; i < inventory.size(); i++) {
```
Comment (larsbjn) [inline src/hospitalgame/item/Inventory.java]: Again, maybe a for each loop?

**Review by larsbjn — CHANGES_REQUESTED** [review]
Maybe use for each loops instead of for loops

**Review by larsbjn — APPROVED** [review]

**Pull request merged**